model/client4.go

```diff
@@ -4239,6 +4239,29 @@ func (c *Client4) uploadPlugin(file io.Reader, force bool) (*Manifest, *Response
 	return ManifestFromJson(rp.Body), BuildResponse(rp)
 }
 
+func (c *Client4) InstallPluginFromUrl(url string) (*Manifest, *Response) {
```

Comment:
I don't see the need to add this method. What do you think about exporting `installPluginFromUrl` instead?

Reply:
Just below this function is another called `InstallPluginFromUrlForced`, which does the same thing, except overwrites an existing plugin with the same id by passing a `true` boolean flag. I used the same pattern that `uploadPlugin` is using with having two exported functions, one with `forced=false` and one with `forced=true`.

@hanzei Does this choice make sense with the explanation?

Reply:
I understand what you did here. Is there another reason for this other then consistency?

3/5 that we should just use parameters to indicate different behaviours of a method.

Reply:
No, mainly just for consistency.

I agree with your perspective, thanks for pointing this out. I'll update to export just the one function.